pinot-common/src/main/java/org/apache/pinot/common/utils/PinotDataType.java

```diff
@@ -342,7 +344,7 @@ public String toString(Object value) {
 
     @Override
     public byte[] toBytes(Object value) {
-      return BytesUtils.toBytes((String) value);
+      return BytesUtils.toBytes(((String) value).trim());
```

Comment:
I feel we only need to trim numbers?

Reply:
String with spaces cannot be parsed as hex-encoded bytes